autozone-poc/src/index.html

```diff
@@ -21,8 +21,16 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Autozone POC</title>
-  </head>
-  <body>
-    <div id="container"></div>
-  </body>
+
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
```

Comment:
remove this code

Reply:
need for jquery element

Reply:
you can handle by installing jquery in node module and logic in   componentdidmount or useEffect method.